fontforge/macbinary.c

```diff
@@ -2577,10 +2571,10 @@ static FOND *PickFOND(FOND *fondlist,char *filename,char **name, int *style) {
     char *find = NULL;
 
     if ((pt = strrchr(filename,'/'))!=NULL ) pt = filename;
-    if ( (lparen = strchr(filename,'('))!=NULL && strchr(lparen,')')!=NULL ) {
+    if ( lparen = SFSubfontnameStart(pt) ) {
+	find = copy(lparen+1);
+	find[strlen(find)-1] = '\0';
 	find = copy(lparen+1);
```

Comment:
This line shows up twice and might cause a memory leak.

Reply:
Removed